test/metrics/NumberMethods/1.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
```

Comment:
Add a disclaimer to all test files. See test/integration/samples/*

Reply:
ok